### ISSUE TITLE
About model checkpoingts and training logs

### ISSUE THREAD

**Issue by sglucas** [MikaStars39/PeRL#33]
Thank you very much for your amazing project. I would like to ask whether you plan to release the model checkpoints and training logs. Thank you!

**Comment by sglucas** [comment]
Thank you @MikaStars39

**Comment by sglucas** [comment]
Hi @MikaStars39 I'm not sure whether it is because the constant learning rate.

**Comment by MikaStars39** [comment]
> Hi [@MikaStars39](https://github.com/MikaStars39) I'm not sure whether it is because the constant learning rate.

Hi, other projects like JustRL [https://github.com/thunlp/JustRL](https://github.com/thunlp/JustRL) also use constant learning rate so I personally believe it works well. Maybe you can try (1 using dapo or (2 turning to a smaller global batch size e.g., from 128->64

**Comment by sglucas** [comment]
Hi @MikaStars39 Thank you for your suggestion and I am trying DAPO. By the way, may I ask your trl and torch version for the experiments.

**Comment by MikaStars39** [comment]
> Hi [@MikaStars39](https://github.com/MikaStars39) Thank you for your suggestion and I am trying DAPO. By the way, may I ask your trl and torch version for the experiments.

I've updated the strict requirements here #28 . You can check the version of trl and vllm.
